checkstyle-tester/diff.groovy

```diff
@@ -289,6 +291,27 @@ def generateSummaryIndexHtml(diffDir, checkstyleBaseReportInfo, checkstylePatchR
     println 'Creating report summary page finished...'
 }
 
+def generateListOfProjectsHtml(diffDir, listOfProjects) {
```

Comment:
I can see us wanting to print other files. We already have an issue for the config files.

Can we make this method more generic, or split it to have 1 generic and 1 specific?

Reply:
Is the other issue for config files #217?  I just want to get a feel for the other inputs.

Reply:
And https://github.com/checkstyle/contribution/issues/430 . #217 might be a bit more complex since it basically requires parsing the config file and know what is a file input. #430 is a parameter passed to the util, so it should be easy to add more support for it once this is merged.